File: NorthWeird.Domain/Entities/Category.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NorthWeird.Domain.Entities
{
    public class Category
    {
        public int CategoryId { get; set; }

        public string CategoryName { get; set; }

        public string Description { get; set; }

        public byte[] Picture { get; set; }
    }
}

File: NorthWeird.ReactClient/src/components/CategoryList/components/Category.js
import React, { Component } from 'react';

class Category extends Component {
  render() {
    const { category: { categoryName, description } } = this.props;
    return (
      <tr>
        <td>
          {categoryName}
        </td>
        <td>
          {description}
        </td>
      </tr>
    );
  }
}

export default Category;
